pkg/operator/ceph/csi/template/nfs/csi-nfsplugin-provisioner-dep.yaml

```diff
@@ -21,6 +21,44 @@ spec:
       priorityClassName: {{ .ProvisionerPriorityClassName }}
       {{ end }}
       containers:
+        - name: csi-snapshotter
```

Comment:
put this behind a flag like rbd and cephfs. we need to provide an option for user to disable it in some cases